plugins/cisco_umbrella_destinations/plugin.spec.yaml

```
@@ -0,0 +1,534 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: cisco_umbrella_destinations
+title: Cisco Umbrella Destinations
+version: 1.0.0
+description: This plugin utilizes Cisco Umbrella to get the most complete view of the relationships
+  and evolution of Internet domains, IP addresses, and autonomous systems to pinpoint
+  attackers infrastructures and predict future threats
+vendor: rapid7
+support: rapid7
+status: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/cisco_umbrella_destinations
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.cisco.com
+tags:
+- cisco
+- umbrella
+- destinations
+hub_tags:
+  use_cases: [threat_detection_and_response]
+  keywords: [cisco, umbrella, destinations]
```

```suggestion
  keywords: [cisco, umbrella]
```
destinations is now an unsupported keyword I guess haha - not sure what it might have been changed to